### PR TITLE
Puppet scripts for AU region demo.

### PR DESCRIPTION
**Pull request by nizzlo**
## Automation tests
 - Unit tests 
   > Passed
 - Integration tests
   > Passed

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: modules/obam/manifests/init.pp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #----------------------------------------------------------------------------
-#  Copyright (c) 2019 WSO2, Inc. http://www.wso2.org
+#  Copyright (c) 2021 WSO2, Inc. http://www.wso2.org
```
Comment (VimukthiPerera) [inline modules/obam/manifests/init.pp:2]: @nizzlo Do not change the copyright notice of the existing files. You need to add the copyright notice with the year the file is added to the repo.

File: modules/obam/manifests/obam_demo.pp [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    source => 'puppet:///modules/obam/com.wso2.finance.open.banking.token.generator-1.5.0.jar',
+}
+
+file { $obam_open_banking_common:
+    mode => "0644",
+    owner => 'root',
+    group => 'root',
+    source => 'puppet:///modules/obam/com.wso2.finance.open.banking.common-1.5.0.jar',
+}
+
```
Comment (VimukthiPerera) [inline modules/obam/manifests/obam_demo.pp:71]: Please format the code.

File: modules/obkm/manifests/obkm_au.pp [hunk 3]
```diff
@@ -0,0 +1,67 @@
+#----------------------------------------------------------------------------
+#  Copyright (c) 2020 WSO2, Inc. http://www.wso2.org
```
Comment (VimukthiPerera) [inline modules/obkm/manifests/obkm_au.pp:2]: This is a new file. Therefore the copyright notice  should be updated.

File: modules/obkm/manifests/obkm_au.pp [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  match   => '<ClientAuthenticationHeader>',
+}
+
+file_line {'Updating identity.xml to display the DCR endpoint in OpenID Connect Discovery, identity.xml' :
+  path  => $obkm_identity,
+  line   => "<OAuth2DCREPUrl>\${carbon.protocol}://${apim_hostname}:8243/open-banking/0.1/register</OAuth2DCREPUrl>",
+  match => '<OAuth2DCREPUrl>',
+  append_on_no_match => false,
+}
+
```
Comment (VimukthiPerera) [inline modules/obkm/manifests/obkm_au.pp:66]: Please format the code.

**Review by VimukthiPerera — CHANGES_REQUESTED** [review]

File: modules/obam/manifests/init.pp (hunk 1 above)
Comment (nizzlo) [reply modules/obam/manifests/init.pp]: fixed them

File: modules/obam/manifests/obam_demo.pp (hunk 2 above)
Comment (nizzlo) [reply modules/obam/manifests/obam_demo.pp:64]: fixed them

File: modules/obkm/manifests/obkm_au.pp (hunk 3 above)
Comment (nizzlo) [reply modules/obkm/manifests/obkm_au.pp:2]: fixed them

File: modules/obkm/manifests/obkm_au.pp (hunk 4 above)
Comment (nizzlo) [reply modules/obkm/manifests/obkm_au.pp:66]: fix them

File: modules/obkm/manifests/obkm_au.pp (hunk 3 above)
Comment (VimukthiPerera) [reply modules/obkm/manifests/obkm_au.pp:2]: Should be 2021.

**Review by VimukthiPerera — APPROVED** [review]

**Review by lalaji — APPROVED** [review]

**Pull request merged**